packages/db/src/migrations/20240622005304_add_levels_fyi.ts

```diff
@@ -0,0 +1,15 @@
+import { type Kysely } from 'kysely';
+
+export async function up(db: Kysely<any>) {
+  await db.schema
+    .alterTable('companies')
+    .addColumn('levels_fyi', 'text')
+    .execute();
+}
+
+export async function down(db: Kysely<any>) {
+  await db.schema
+    .alterTable('companies')
+    .dropColumn('levels_fyi')
+    .execute();
+}
```

Comment:
It looks like you might be missing the VSCode extension [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode). You can download this within VSCode and then it should fix the formatting of this file (which is what's giving the error) upon saving